tests/testthat/test-scale_colour_2dii.R

```diff
@@ -100,3 +98,53 @@ test_that("warns about assigning colours", {
     regexp = "Assigning colours to unrecognised names in data"
   )
 })
+
+test_that("scale_*_colour is sensitive to `labels`", {
+  new_labels <- c("lab1", "lab2", "lab3", "lab4")
+
+  p <- example_plot_scale_colour() +
+    scale_colour_2dii(labels = new_labels)
+  g <- ggplot2::ggplot_build(p)
+
+  expect_true(identical(g$plot$scales$scales[[1]]$labels, new_labels))
+})
+
+test_that("scale_*_fill is sensitive to `labels`", {
+  new_labels <- c("lab1", "lab2", "lab3", "lab4")
+
+  p <- example_plot_scale_fill() +
+    scale_fill_2dii(labels = new_labels)
+  g <- ggplot2::ggplot_build(p)
+
+  expect_true(identical(g$plot$scales$scales[[1]]$labels, new_labels))
+})
+
+test_that("scale_*_colour works with different letter cases", {
+  p1 <- ggplot2::ggplot(example_data_lowercase(), ggplot2::aes(x = x, y = val)) +
+    ggplot2::geom_line(ggplot2::aes(colour = variable)) +
+    scale_colour_2dii(palette = "pacta", colour_groups = example_data_lowercase()$variable)
+
+  p2 <- ggplot2::ggplot(example_data_uppercase(), ggplot2::aes(x = x, y = val)) +
+    ggplot2::geom_line(ggplot2::aes(colour = variable)) +
```

Comment:
The calls to `ggplot2::` take a lot of screen real state. I think they obscure the actually interesting code 
:-(

Reply:
I agree. It was required because I didn't have those functions in the `imports`. I left some calls to `ggplot2::` but added `ggplot` and `aes` to imports.